sled-agent/src/bootstrap/server.rs

```diff
@@ -157,18 +132,12 @@ impl Server {
 
 struct Inner {
     listener: TcpListener,
-    sp: Option<SpHandle>,
-    trust_quorum: TrustQuorumMembership,
     bootstrap_agent: Arc<Agent>,
     log: Logger,
 }
 
 impl Inner {
     async fn start_sprockets(
```

Comment:
This should probably get changed from `start_sprockets`, which it no longer even attempts, to `start_bootstrap_agent_listener` or something.